sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -38,15 +38,21 @@
     'PytorchModelHandlerKeyedTensor',
 ]
 
-TensorInferenceFn = Callable[
-    [Sequence[torch.Tensor], torch.nn.Module, str, Optional[Dict[str, Any]]],
-    Iterable[PredictionResult]]
+TensorInferenceFn = Callable[[
+    Sequence[torch.Tensor],
+    torch.nn.Module,
+    torch.device,
+    Optional[Dict[str, Any]],
+    Optional[str]
+],
+                             Iterable[PredictionResult]]
```

Comment:
Nit: What's going on with the spacing here? Can we normalize this a bit?

Reply:
![image](https://user-images.githubusercontent.com/34158215/215150663-99d9e2ff-8706-4c17-95cc-bf4547ff89c2.png)

The formatter is not letting me change it.

Reply:
Weird...